addons/stock/static/src/js/inventory_adjustments_list_view.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('stock.InventoryAdjustmentsListView', function (require) {
```

Comment:
It would be great to have a commit with file rename and another with all the modification (the purpose is to keep git history)